modules/common-fs2/src/test/scala/com/snowplowanalytics/snowplow/enrich/common/fs2/test/TestEnvironment.scala

```diff
@@ -167,6 +167,8 @@ object TestEnvironment extends CatsIO {
                       metrics,
                       metadata,
                       None,
+                      _ => "",
+                      _ => "",
```

Comment:
I guess this could be any string.  Why did you choose the empty string instead of some placeholder?  If you used a distinctive string (e.g. `test_good_partition_key`) then you could assert that Enrich sets the correct key.

Reply:
no reason really, that's a great idea, thanks!